Detectors/ITSMFT/common/reconstruction/include/ITSMFTReconstruction/RUDecodeData.h

```diff
@@ -117,7 +117,14 @@ int RUDecodeData::decodeROF(const Mapping& mp, const o2::InteractionRecord ir, b
     while ((ret = AlpideCoder::decodeChip(*chipData, cableData[icab], *seenChipIDsPtr, chIdGetter)) || chipData->isErrorSet()) { // we register only chips with hits or errors flags set
       setROFInfo(chipData, cableLinkPtr[icab]);
       if (verifyDecoder) {
-        seenChips[chipData->getChipID()] = chipData;
+        auto ID = chipData->getChipID();
+        // Since the errored chips are not recorded, we need to add them
+        // separately
+        if (std::find(seenChipIDsInCable.begin(), seenChipIDsInCable.end(),
+                      ID) == seenChipIDsInCable.end()) {
+          seenChipIDsInCable.push_back(ID);
```

Comment:
Indeed, you are right. Also, it seems we do not need to later sort the `seenChipIDsInCable` vector as well, since the chipIDs are added in the correct order.